app-modules/engagement/resources/views/components/engagement-timeline-item.blade.php

```diff
@@ -8,18 +8,18 @@
     <div class="flex flex-row justify-between">
         <h3 class="mb-1 flex items-center text-lg font-semibold text-gray-500 dark:text-gray-100">
             <a
-                class="font-medium underline"
-                href="{{ UserResource::getUrl('view', ['record' => $record->createdBy]) }}"
+                    class="font-medium underline"
```

Comment:
Can we remove the formatting spacing changes?

Reply:
I can, but this will be fixed automatically once we merge in the auto linting / formatting PR which makes sure all of our specified files are covered.